keras_contrib/losses/segmentation_losses.py

```diff
@@ -0,0 +1,54 @@
+""" Loss functions for semantic segmentation algorithms.
+
+    adapted from: https://github.com/theduynguyen/Keras-FCN
+"""
+from keras import losses
+import keras.backend as K
+import tensorflow as tf
```

Comment:
Is it necessary?

Reply:
nope should be easy to remove, I got it out of all the functions but forgot to remove the import haha